pom.xml

```diff
@@ -103,7 +104,6 @@
         <url>https://github.com/opensearch-project/security</url>
         <connection>scm:git:git@github.com:opensearch-project/security.git</connection>
         <developerConnection>scm:git:git@github.com:opensearch-project/security.git</developerConnection>
-        <tag>1.0.1.0</tag>
```

Comment:
Why are we removing the tag here?

Reply:
What is the purpose of it here? I believe it's unused.

Reply:
`<url>`, `<connection>` and `<developerConnection>` are not used either. It is there mostly for historical reasons. I don't see a good reason to remove `<tag>` only.